README.md

```diff
@@ -85,43 +85,57 @@ Shows and Movies are sourced from Streaming Sites.
 
 ### Prerequisites
 
-- [`mpv`](https://mpv.io) - player used for Windows, Linux and Android
-- [`iina`](https://iina.io) - player used for MacOS
-- [`Outplayer`](https://outplayer.app/) - player used for iOS
-- [`ffmpeg`](https://github.com/FFmpeg/FFmpeg) - For downloads 
-- [`fzf`](https://github.com/junegunn/fzf) - The selection Menu
+To get started, make sure you have the following installed:
+
+- [`mpv`](https://mpv.io) - Player for Windows, Linux, and Android
+- [`iina`](https://iina.io) - Player for MacOS
+- [`Outplayer`](https://outplayer.app/) - Player for iOS
+- [`ffmpeg`](https://github.com/FFmpeg/FFmpeg) - Media encoder
+- [`fzf`](https://github.com/junegunn/fzf) - Selection Menu
 
 
 ## Installation
-<!-- WIN AND LINUX -->
-### Windows / Linux
-  - Run this Command inside your Terminal
+<!-- WINDOWS -->
+### Windows
+(Recommended) Run the following commands (including mov-cli) on any Windows system using the latest modern build version of cmd from the Microsoft Store: [terminal](https://apps.microsoft.com/detail/9n0dx20hk701?hl=en-us&gl=US)
+  - Run as admin:
     ``` 
-    pip install mov-cli
+    powershell -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))"  
+    ```
+  - Restart cmd as admin, then run:
     ```
-  - Optional 
+    choco install -y python3
+    choco install -y ffmpeg-full
+    choco install -y fzf
+    choco install -y mpv
+    ```
```

Comment:
Nitpick but can't these packages be installed on one line like this:
```sh
choco install python3 ffmpeg-full fzf mpv
```
Btw I've never used chocolatey so correct me if I'm wrong.

Reply:
Yess, you can, and I totaly forgot about it . Thank u for calling this out